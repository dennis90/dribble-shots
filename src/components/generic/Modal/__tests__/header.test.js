import React from 'react'
import {shallow} from 'enzyme'

import Header from '../header'

describe('ModalHeader Component', () => {
  it('should render div with close button', () => {
    const wrapper = shallow(<Header closeRoute={'/'} />)
    expect(wrapper).toMatchSnapshot()
  })
})
