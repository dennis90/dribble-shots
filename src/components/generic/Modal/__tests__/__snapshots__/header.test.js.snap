// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalHeader Component should render div with close button 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    closeRoute="/"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        replace={false}
        to="/"
>
        <span
                className="close-modal-button"
        />
</Link>,
      "className": "modal-header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "children": <span
          className="close-modal-button"
/>,
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "className": "close-modal-button",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          replace={false}
          to="/"
>
          <span
                    className="close-modal-button"
          />
</Link>,
        "className": "modal-header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "children": <span
            className="close-modal-button"
/>,
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "className": "close-modal-button",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
